app/vue/tsconfig.json

```diff
@@ -3,7 +3,9 @@
   "compilerOptions": {
     "rootDir": "./src",
     "types": ["webpack-env"],
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "strict": true,
+    "noImplicitReturns": true
```

Comment:
Made the Vue app as strict as I could for typescript, to catch everything, can revert these if wanted.